syntax/toml.vim

```diff
@@ -46,7 +46,10 @@ syn match tomlDate /\d\{2\}:\d\{2\}:\d\{2\}\%(\.\d\+\)\?/ display
 syn match tomlDate /\d\{4\}-\d\{2\}-\d\{2\}[T ]\d\{2\}:\d\{2\}:\d\{2\}\%(\.\d\+\)\?\%(Z\|[+-]\d\{2\}:\d\{2\}\)\?/ display
 hi def link tomlDate Constant
 
-syn match tomlKey /\v(^|[{,])\s*\zs[[:alnum:]._-]+\ze\s*\=/ display
+syn match tomlDottedKey /\v[^.]+\zs\./ contained display
```

Comment:
`tomlDottedKey` should really be `tomlDotInKey` because it matches the dot not the whole dotted key.

Reply:
Makes sense to me! I've changed that, rebased, and force-pushed since there were merge conflicts.